circuit-benchmarks/src/evm_circuit.rs

```diff
@@ -13,6 +13,14 @@ pub struct TestCircuit<F> {
     block: Block<F>,
 }
 
+impl<F: Default> TestCircuit<F> {
+    pub fn new() -> TestCircuit<F> {
+        let mut test_circuit = TestCircuit::default();
+        test_circuit.block = Block::new();
+        test_circuit
+    }
+}
```

Comment:
With `TestCircuit::new` I think we can also remove the `Default` derive of `TestCircuit`.